This file is part of MXE.
See index.html for further information.

Contains ad hoc patches for cross building.

From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Raoul Hecky <r.hecky@emc-partner.ch>
Date: Thu, 28 Jul 2016 15:55:26 +0200
Subject: [PATCH] Fix qguiapplication.h include path when using angle+mingw on
 linux


diff --git a/src/plugins/common/evr/evrd3dpresentengine.cpp b/src/plugins/common/evr/evrd3dpresentengine.cpp
index 1111111..2222222 100644
--- a/src/plugins/common/evr/evrd3dpresentengine.cpp
+++ b/src/plugins/common/evr/evrd3dpresentengine.cpp
@@ -49,7 +49,7 @@
 #include <private/qmediaopenglhelper_p.h>
 
 #ifdef MAYBE_ANGLE
-# include <qtgui/qguiapplication.h>
+# include <qguiapplication.h>
 # include <qpa/qplatformnativeinterface.h>
 # include <qopenglfunctions.h>
 # include <EGL/eglext.h>
